Assets/Resources/Settings/Network Settings.asset

```diff
@@ -12,9 +12,16 @@ MonoBehaviour:
   m_Script: {fileID: 11500000, guid: faafeaf47c7c1f445be012015dfe565f, type: 3}
   m_Name: Network Settings
   m_EditorClassIdentifier: 
+  lobbyName: bababooeyLobby
```

Comment:
what is the meaning of this ? 😠

Reply:
LOL. Je sais pas, c'est là depuis le début (gracieuseté de Antoine). Malgré ce que Git dit, je l'ai pas ajouté dans cette pr

Reply:
Le nom du lobby photon.